src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetBuilders/DataHistoryWidgetBuilder/DataHistoryWidgetBuilder.tsx

```diff
@@ -132,7 +134,8 @@ const DataHistoryWidgetBuilder: React.FC<IDataHistoryWidgetBuilderProps> = ({
     }, [adxConnectionInformation]);
 
     const connectionString = formData.widgetConfiguration.connection
-        ? generateConnectionString(formData.widgetConfiguration.connection)
+        ? generateConnectionString(formData.widgetConfiguration.connection) ||
```

Comment:
how does this look? Is this the value in the input box or some description next to it?